# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde/kdirstat/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual gcc-3.4 fixes. To those removing of -Wmissing-prototypes belongs,
but since those are spread over so many files I use th gcc wrapper for this.

  - Rene Rebe <rene@exactcode.de>

--- ./kdirstat/kdirstatapp.h.orig	2003-08-26 22:47:47.000000000 +0200
+++ ./kdirstat/kdirstatapp.h	2004-05-07 19:52:50.088330112 +0200
@@ -46,7 +46,7 @@
     class KSettingsDialog;
     class KTreemapView;
     class KTreemapTile;
-};
+}
 
 using namespace KDirStat;
 
